lib/rubocop/extension/generator/generator.rb

```diff
@@ -98,6 +98,7 @@ def self.defaults!
           patch "lib/#{dirname}.rb", 'module Rubocop', 'module RuboCop'
           patch "lib/#{dirname}/version.rb", 'module Rubocop', 'module RuboCop'
           patch "#{name}.gemspec", 'Rubocop', 'RuboCop'
+          2.times { patch "spec/#{dirname}_spec.rb", 'Rubocop::', 'RuboCop::' }
```

Comment:
Oh, I guess this is an actual bug fix, not just a smoke test fix. The smoke test _found_ the bug, because it would blow up due to the constant being incorrect. See commit message for details.